container/karaf/src/main/filtered-resources/etc/custom.properties

```diff
@@ -74,6 +78,39 @@ org.osgi.framework.system.packages.extra=org.apache.karaf.branding,\
         com.ibm.wsdl.util;version=${wsdl4jVersion},\
         com.ibm.wsdl.xml;version=${wsdl4jVersion},\
         com.ibm.wsdl;version=${wsdl4jVersion},\
+        com.sun.activation.registries;version=1.2.1,\
```

Comment:
They _are_ provided in `$OPENNMS_HOME/lib` and I'm loathe to take it out since there will be plenty of things still using it on the Karaf side too, and I don't want to hit weird cross-classpath issues.

Eventually when we move to JAXB 3 then everything will be in the new `javax.activation` namespace and everything should be fine.  This stuff is all weird because we're riding a line between older JAXB support and newer stuff that assumes jdk9+ `module`s and such.